---
layout: page
title: Publications
permalink: /publications/
---
For citations and other metrics, please see [Google scholar](https://scholar.google.com/citations?user=PrYu53UAAAAJ&hl=en&authuser=2). 
## 2023

Santos, Fox, Mohan, O'Malley, Viswanathan, Lubbers. [Development of the Senseiver for efficient field reconstruction from sparse observations](https://www.nature.com/articles/s42256-023-00746-x#citeas). *Nature Machine Intelligence*. 

Blanchard, Bhowmik, Fox, Gounley, Glaser, Akpa, Irle. [Adaptive language model training for molecular design](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-023-00719-7). *Journal of Cheminformatics*. 

Fox, English, Akpa. [Targeting tissues via dynamic human systems modelin in generative design](https://openreview.net/attachment?id=lOIKlYR3vX&name=pdf).*GenBio Workshop, NeurIPS*. 

Fox, Ghosh. [Active Causal Machine Learning for Molecular Property Prediction](https://openreview.net/forum?id=57EslEJNOj). *AI4Mat Workshop, NeurIPS.*

Fox, Batt, Ruess. [Bayesian filtering for model predictive control of stochastic gene expression in single cells](https://iopscience.iop.org/article/10.1088/1478-3975/ace094). *Physical Biology*. 

Santos, Fox, Lubbers, Lin. [Blackout Diffusion: Generative Diffusion Models in Discrete-State Spaces](https://arxiv.org/abs/2305.11089). *ICML*. 

Spannaus, Gounley, Chandra Shekar, Fox, Mohd-Yusof, Schaefferkoetter, Hanson. [FrESCO: Framework for Exploring Scalable Computational Oncology](https://joss.theoj.org/papers/10.21105/joss.05345). *Journal of Open Source Software*. 

Raymond, Ghaffari, Aguilera, Ron, Morisaki, Fox, May, Stasevich, Munsky. [Using Mechanistic Models and Machine Learning to Design Single-Color Multiplexed Nascent Chain Tracking Experiments](https://www.frontiersin.org/articles/10.3389/fcell.2023.1151318/full). *Frontiers in Cell Dev. Bio*.


## 2022 
(Fox, Fletcher), Fraisse, Aditya, Sosa-Carrilo, Petit, Gilles, Bertaux, Ruess, Batt. [Enabling reactive microscopy with MicroMator](https://www.nature.com/articles/s41467-022-29888-z#citeas). *Nature Communications.*

Santos, Fox, Mohan, Viswanathan, Lubbers. [The Senseiver: attention-based global field reconstruction from sparse observations](https://ml4physicalsciences.github.io/2022/files/NeurIPS_ML4PS_2022_173.pdf). Neurips workshop on Machine Learning and the Physical Sciences.

Fox. [Extracting Information from Stochastic Trajectories of Gene Expression](https://arxiv.org/pdf/2206.14874.pdf). *arXiv preprint*. 

## 2021
Fox, Barkai, Krapf. [Aging power spectrum of membrane protein transport and other subordinated random walks](https://www.nature.com/articles/s41467-021-26465-8). *Nature Communications*.

Jashnsaz, Fox, Munsky, Neuert. [Building predictive signaling models by perturbing yeast cells with time-varying stimulations resulting in distinct signaling responses](https://star-protocols.cell.com/protocols/786). *STAR Protocols.*


## 2020 
Jashnsaz, Fox, Hughes, Li, Munsky, Neuert. [Diverse cell stimulation kinetics identify predictive signal transduction models](https://www.cell.com/iscience/fulltext/S2589-0042(20)30757-4). *iScience.*

Fox, Neuert, Munsky. [Optimal Design of Single-Cell Experiments within Temporally Fluctuating Environments](https://www.hindawi.com/journals/complexity/2020/8536365/). *Complexity.* 

## 2019

Aguilera, Raymond, Fox, May, Djokic, Morisaki, Stasevich, Munsky. [Computational design and interpretation of single-RNA translation experiments](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007425). *PloS Computational Biology.*

Fox and Munsky. [The finite state projection based Fisher information matrix approach to estimate and maximize the information in single-cell experiments](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006365). *PloS Computational Biology.*

Vo, Fox, Baetica, Munsky. [Bayesian estimation for stochastic gene expression using multifidelity models](https://pubs.acs.org/doi/10.1021/acs.jpcb.8b10946). *Journal of Physical Chemistry B.*

## 2018 

Munsky, Li, Fox, Shepherd, Neuert. [Distribution shapes govern the discovery of predictive models for gene regulation](http://www.pnas.org/content/early/2018/06/28/1804060115.short). *Proceedings of the National Academy of Sciences.*
## 2017
Fox and Munsky. [Stochasticity or Noise in Biochemical Reactions](https://arxiv.org/abs/1708.09264). *arXiv preprint.*
## 2016 
Fox, Neuert, Munsky. [Finite state projection based bounds to compare chemical equation models using single-cell data](https://aip.scitation.org/doi/full/10.1063/1.4960505). *Journal of Chemical Physics.* 
## 2015 
Munsky, Fox, Neuert. [Integrating single-molecule experiments and discrete stochastic models to understand heterogeneous gene trancsription dynamics](https://www.sciencedirect.com/science/article/pii/S1046202315002510). *Elsevier Methods .*

Soltani, Bokes, Fox, Singh. [Nonspecific transcription factor binding can reduce noise in the expression of downstream proteins](http://iopscience.iop.org/article/10.1088/1478-3975/12/5/055002/meta). *Physical biology.*

N Rangarajan, Fox, Singh, Kulkami, G Rangarajan. [Disorder, oscillatory dynamics and state switching: the role of c-Myc](https://www.sciencedirect.com/science/article/pii/S0022519315004567). *Journal of Theoretical Biology.*
## 2014
Fox and Singh. [Stochastic analysis of protein-mediated and microRNA-mediated feedback circuits in HIV](https://www.sciencedirect.com/science/article/pii/S1474667016417684). *IFAC Proceedings Volumes 47.* 

